web/views/goal_view.ex

```diff
@@ -1,9 +1,14 @@
 defmodule Healthlocker.GoalView do
   use Healthlocker.Web, :view
+  use Timex
```

Comment:
No worries, no point using it if we dont need to. But I'd just remove this unused import.